client/blocks/reader-combined-card/index.jsx

```diff
@@ -0,0 +1,72 @@
+/**
+ * External Dependencies
+ */
+import React from 'react';
+import { get } from 'lodash';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal Dependencies
+ */
+import Card from 'components/card';
+import { getStreamUrl } from 'reader/route';
+import ReaderAvatar from 'blocks/reader-avatar';
+import ReaderSiteStreamLink from 'blocks/reader-site-stream-link';
+import { siteNameFromSiteAndPost } from 'reader/utils';
+import ReaderCombinedCardPost from './post';
+
+const ReaderCombinedCard = ( { posts, site, feed, onClickPost, translate } ) => {
+	const feedId = get( feed, 'ID' );
+	const siteId = get( site, 'ID' );
+	const siteIcon = get( site, 'icon.img' );
+	const feedIcon = get( feed, 'image' );
+	const streamUrl = getStreamUrl( feedId, siteId );
+	const siteName = siteNameFromSiteAndPost( site, posts[ 0 ] );
+
+	return (
+		<Card className="reader-combined-card">
+			<header className="reader-combined-card__header">
+				<ReaderAvatar
+					siteIcon={ siteIcon }
+					feedIcon={ feedIcon }
+					author={ null }
```

Comment:
We don't have an author :)

In the combined card header, we only ever show the site or feed icon.